Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -151,6 +166,56 @@ public TestsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
             Assert.Equal(0, _trappistDbContext.Test.Count());
         }
 
+        /// <summary>
+        /// Adds the selected category to TestCategory
+        /// </summary>      
+        [Fact]
+        public async Task AddTestCategory()
+        {
+            var category = CreateCategory();
+            await _categoryRepository.AddCategoryAsync(category);
+            var test = CreateTest("Final");
+            await _testRepository.CreateTestAsync(test);
+            TestCategory testcategory = new TestCategory();
+            testcategory.TestId = test.Id;
+            testcategory.CategoryId = category.Id;
+            List<TestCategory> testCategoryList = new List<TestCategory>();
+            testCategoryList.Add(testcategory);
+            var testAC = await _testRepository.GetTestDetailsByIdAsync(test.Id);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused 'testAC' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)